Website/views/pages/admin.ejs

```diff
@@ -41,6 +41,14 @@
             <input type="submit" value="Reset Tables">
         </form>
 
+        <form method="get" action="/logs">
```

Comment:
It would be nice to have these float to the right so they button group isn't so dense

Reply:
Or have the related log buttons side by side

Reply:
Done